packages/jellyfish-api-core/__tests__/category/oracle/listLatestRawPrices.test.ts

```diff
@@ -0,0 +1,255 @@
+
+    // NOTE(jingyi2811): Set to 1 second after 1 hour before.
+    const timestamp4 = Math.floor(new Date().getTime() / 1000) - 3599
+    const prices4 = [{ tokenAmount: '2.0@APPLE', currency: 'EUR' }]
+    await container.call('setoracledata', [oracleid4, timestamp4, prices4])
+
+    // NOTE(jingyi2811): Set to 1 hour before.
+    const timestamp5 = Math.floor(new Date().getTime() / 1000) - 3600
+    const prices5 = [{ tokenAmount: '2.5@APPLE', currency: 'EUR' }]
+    await container.call('setoracledata', [oracleid5, timestamp5, prices5])
+
+    await container.generate(1)
+
+    // NOTE(jingyi2811): Pagination is not supported.
+    const data = await client.oracle.listLatestRawPrices()
+
+    expect(data.length).toStrictEqual(5)
+
+    const result1 = data.find(element => element.oracleid === oracleid1)
+
+    expect(result1).toStrictEqual(
+      {
+        priceFeeds: priceFeed,
+        oracleid: oracleid1,
+        weightage: new BigNumber(1),
+        timestamp: new BigNumber(timestamp1),
+        rawprice: new BigNumber(0.5),
+        state: OracleRawPriceState.LIVE
+      })
+
+    const result2 = data.find(element => element.oracleid === oracleid2)
+
+    expect(result2).toStrictEqual(
+      {
+        priceFeeds: priceFeed,
+        oracleid: oracleid2,
+        weightage: new BigNumber(2),
+        timestamp: new BigNumber(timestamp2),
+        rawprice: new BigNumber(1),
+        state: OracleRawPriceState.LIVE
+      })
+
+    const result3 = data.find(element => element.oracleid === oracleid3)
+
+    expect(result3).toStrictEqual(
+      {
+        priceFeeds: priceFeed,
+        oracleid: oracleid3,
+        weightage: new BigNumber(3),
+        timestamp: new BigNumber(timestamp3),
+        rawprice: new BigNumber(1.5),
+        state: OracleRawPriceState.EXPIRED
+      })
+
+    const result4 = data.find(element => element.oracleid === oracleid4)
+
+    expect(result4).toStrictEqual(
+      {
+        priceFeeds: priceFeed,
+        oracleid: oracleid4,
+        weightage: new BigNumber(4),
+        timestamp: new BigNumber(timestamp4),
+        rawprice: new BigNumber(2),
+        state: OracleRawPriceState.LIVE
+      })
+
+    const result5 = data.find(element => element.oracleid === oracleid5)
+
+    expect(result5).toStrictEqual(
+      {
+        priceFeeds: priceFeed,
+        oracleid: oracleid5,
+        weightage: new BigNumber(5),
+        timestamp: new BigNumber(timestamp5),
+        rawprice: new BigNumber(2.5),
+        state: OracleRawPriceState.EXPIRED
+      })
+
+    await container.call('removeoracle', [oracleid1])
+    await container.call('removeoracle', [oracleid2])
+    await container.call('removeoracle', [oracleid3])
+    await container.call('removeoracle', [oracleid4])
+    await container.call('removeoracle', [oracleid5])
+
+    await container.generate(1)
+  })
+
+  it('should listLatestRawPrices with empty array if there is no oracle appointed', async () => {
```

Comment:
you should test 1 more condition when you set 2 feed. and listLatestRawPrices with 1 feed making sure only 1 feed is returned.